guides/deploy/starter_deploy_cloudfoundry.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+
+<head>
+  <title>Learn - Deploy using Cloud Foundry</title>
+  <link rel="icon" type="image/png" href="../../assets/favicon-32x32.png" sizes="32x32" />
+  <link rel="icon" type="image/png" href="../../assets/favicon-16x16.png" sizes="16x16" />
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="../../scripts/prism.js"></script>
+  <link rel="stylesheet" href="../../css/reset.css">
+  <link rel="stylesheet" href="../../css/main.css">
+  <link rel="stylesheet" href="../../css/guides.css">
+  <link rel="stylesheet" media="screen and (max-width: 900px)" href="../../css/mobile_guides.css">
+</head>
+
+<header>
+  <div class="header-container">
+    <div class="header-main">
+      <a class="home-link" href="../../index.html">
+        <img class="header-logo" src="../../assets/kitura-logo.png" alt="Kitura logo">
+        <h1 class="header-title">KITURA</h1>
+      </a>
+    </div>
+    <nav class="header-nav">
+      <a class="header-link active-nav" href="../../learn.html">LEARN</a>
+      <a class="header-link" href="../../packages.html">PACKAGES</a>
+      <a class="header-link" href="../../events.html">EVENTS</a>
+      <a class="header-link" href="../../help.html">HELP</a>
+    </nav>
+  </div>
+</header>
+<body>
+  <section class="guide-content">
+    <div class="title-block">
+        <img width="480px" src="../../assets/Kitura.svg" alt="Kitura Logo">
+      <h1>DEPLOY YOUR APP USING CLOUD FOUNDRY</h1>
+    </div>
+
+<p>This guide will show you how to deploy the Kitura application you created by following our <a href="https://www.kitura.io/guides/gettingstarted">Getting Started</a> guide to a <a href="#cloud-foundry">Cloud Foundry</a> based cloud environment. The guide assumes you already have an environment to deploy to. If not, you can <a href="https://console.bluemix.net/registration/">sign up</a> to IBM Cloud for free and use it to host your Cloud Foundry apps.</p>
+<h4 >Starter project contents</h4>
+<p>Your generated Kitura project already contains a manifest.yml file which you can use to deploy your application. This file is optional, but allows you to maintain consistency across deployments. If necessary you can adjust or add attributes, a list of which can be found <a href="https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#optional-attributes">here</a>. You’ll use the Cloud Foundry command line interface, which you can install using the instructions <a href="https://github.com/cloudfoundry/cli#downloads">here</a>.</p>
```

Comment:
How about...

Your generated Kitura project contains a manifest.yml file which you can use to deploy your application. Although you can deploy applications without a manifest, manifests provide consistency and reproducibility across deployments. You can also specify <a href="https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#optional-attributes"> additional application attributes</a> within your manifest.